src/test/java/com/easypost/AddressTest.java

```diff
@@ -109,7 +109,7 @@ public void testCreateVerify() throws EasyPostException {
 
         assertTrue(address instanceof Address);
         assertTrue(address.getId().startsWith("adr_"));
-        assertEquals("417 MONTGOMERY ST STE 500", address.getStreet1());
```

Comment:
These should be `FL 5`, please correct the spelling of `streat` in the fixture itself and re-record cassettes.

Reply:
I have updated the fixture data. There is no VCR in Java CL yet, E2E tested.